examples/ssh_key/main.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_providers {
+    hivelocity = {
+      versions = ["0.0.1"]
+      source = "hivelocity.net/prod/hivelocity"
+    }
+  }
+}
+
+resource "hivelocity_ssh_key" "demo" {
+  name = "My Terraform SSH Key"
+  public_key = "My Terraform SSH Public Key"
```

Comment:
probably want to put a real pubic key here, just to test the real deal

Reply:
Done!